app/components/blacklight/document/bookmark_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<%- # we don't use the per-form csrf token -%>
```

Comment:
I don't love just overriding component templates.  I'd prefer we create a new component that extends the original.  e.g.  `TrackingBookmarkComponent < Blacklight::Document::BookmarkComponent`. That gives us some way to add tests for the new behavior.  It also helps explain to the reader why we have a separate component and what the differences are from the original.